compute_endpoint/tests/integration/endpoint/endpoint/test_endpoint_manager.py

```diff
@@ -444,36 +444,52 @@ def test_delete_endpoint(self, mocker, dir_exists, web_svc_ok, force):
         config_dir = pathlib.Path("/some/path/mock_endpoint")
         ep_uuid_str = str(uuid.uuid4())
 
-        mock_client = mocker.patch(f"{_MOCK_BASE}Client")
         mock_stop_endpoint = mocker.patch.object(Endpoint, "stop_endpoint")
         mock_rmtree = mocker.patch.object(shutil, "rmtree")
-        mocker.patch.object(Endpoint, "get_endpoint_id", return_value=ep_uuid_str)
-
-        # Immediately exit if the dir doesn't exist
-        if not dir_exists:
-            with pytest.raises(SystemExit):
-                manager.delete_endpoint(config_dir, None, force)
-            return
```

Comment:
I would leave this and add `and not force` to the `if` statement.

Reply:
Added back in

Reply:
Actually, left this removed.  As checking the directory itself is done in the decorator for the click CLI argument, it is no longer done in Endpoint.delete_endpoint(), so it should no longer SystemExit immediately if the directory doesn't exist. 

 If the directory doesn't exist, delete_endpoint() merely skips the last ``shutil.rmtree()`` as there isn't really a reason to error out on the last step after the web-service deletion is done.  (the directory should exist in normal scenarios as if given a UUID as argument and it got here, as the dir would have come from endpoint.json in an existing directory).